### ISSUE TITLE
Problems with torrents list display

### ISSUE THREAD

**Issue by Tockman** [transmission/transmission#4767]
### What is the issue?

Pause and video distortions plus very high processor load. Never observed such defect on betas.
![Снимок экрана 2023-02-09 в 11 35 33](https://user-images.githubusercontent.com/74548419/217760588-e59e05b0-4d33-4e55-8362-a5a12cba4139.png)

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.0 (280ace12f8)

**Comment by Coeur** [comment]
I was surprised that the "No Group" label was missing on your screenshot, but then I found the Russian translation:
https://github.com/transmission/transmission/blob/b7099270b6bbcd9181b5b42da1e5a39ccdf182ae/macosx/ru.lproj/Localizable.strings#L551

Anyway, that's probably unrelated.

May I know your version of macOS for reference?
Did the issue happen again a day after, or was it a one time only issue?

**Comment by Tockman** [comment]
1. Yes, I have groups
2. macOS Ventura 13.2
3. As I described it - just one time. But system messages of "Transmission app stops responding" are regular.

**Comment by Tockman** [comment]
It happened again right now.

**Comment by Coeur** [comment]
OK. So, compared to my setup, and ignoring the language difference, you're using two display options which I don't normally use:
- "Pieces Bar" ("Части")
- "Availability" ("Доступность")

Maybe one of those is unstable?

**Comment by Tockman** [comment]
Yes, I use them. Maybe but they are necessary for me. Gonna try to live a day without 'em.

**Comment by Tockman** [comment]
No, they are not a reason. I saw some freezes with Pieces and Availability disabled.

**Comment by Coeur** [comment]
<s>Let's assume this was addressed by #7760 refactor that landed in 4.1.0-beta.4.</s> (never mind, I mixed up with TorrentTableView)

Nonetheless, that's the only report about this issue in more than 2 years, and I never saw it happened (**although I don't use Groups myself**).

If you reproduce in newer builds, kindly tell us in which version?
